js-apps/meep-frontend/src/js/containers/exec/dashboard-cfg-detail-pane.js

```diff
@@ -145,6 +149,104 @@ class DashCfgDetailPane extends Component {
     this.onUpdateDashCfg(DASH_CFG_START_TIME, '', null);
   }
 
+  // Callback function for postSeqQuery
+  postSeqQueryCb(error, data, format) {
+    if (error !== null) {
+      this.refreshSeqMetrics = true;
```

Comment:
No need to set 'refreshSeqMetrics' here.

Reply:
Addressed.